lib/msal-common/src/auth/authority/AuthorityFactory.ts

```diff
@@ -20,14 +31,14 @@ export class AuthorityFactory {
         const authorityUrl = new UrlString(authorityString);
         const components = authorityUrl.getUrlComponents();
         const pathSegments = components.PathSegments;
-        switch (pathSegments[0]) {
-            case "tfp":
-                // tfp denotes a b2c url
-                return AuthorityType.B2C;
-            default:
-                // default authority is always AAD
-                return AuthorityType.Aad;
-        }
+
+        if (pathSegments[0] === "adfs")
```

Comment:
Should this be case insensitive comparison.

Reply:
Yes, good catch. Will update